crowdgezwitscher/events/models.py

```diff
@@ -36,3 +36,21 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('events:detail', kwargs={'pk': self.pk})
+
+    def _is_ready_for_twitter(self):
+        """Checks that all fields required for getting tweets have some value."""
+        return all([self.twitter_account_names, self.twitter_hashtags, self.coverage_start, self.coverage_end])
```

Comment:
Die Eingabe von Hashtags soll optional sein. Account-Namen sind immer erforderlich.
Wenn du das heute Abend ändert reicht das. Ich hab noch ein paar Sachen in Angular zu fixen.
Bitte noch einen Test dafür hinzufügen.